flow-typed/types/prebid-config.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+declare type KPPrebidConfig = {
+  libUrl: string,
+  disable: boolean,
```

Comment:
why is needed?

Reply:
Cause I want to set default values for prebid like timeout and stuff, it could handle differently

Reply:
handled inside the manager for the default timeout valuse